site/assets/scss/_component-examples.scss

```diff
@@ -359,10 +359,11 @@
   }
 
   pre {
-    padding: 0;
-    margin-top: .625rem;
+    padding: .2rem 0 .9rem;
```

Comment:
**question** Is there a reason for these not to be multiples of 1/16th to have better chances to end on round pixels ? (eg. 0.1875/0.25 and 0.875/0.9375)

Reply:
nothing specific other than "cleaner-looking numbers"

Reply:
also, as the margin/padding here is for a box containing text (rather than, say, an image or whatever where hitting full pixels is more important), i think we've decided not to sweat over these sorts of things too much